src/PwdResetEmailSentController.js

```diff
@@ -40,6 +40,17 @@ function (Okta, Enums, FormController, FormType) {
       attributes: { 'data-se': 'pwd-reset-email-sent' },
       formChildren: function () {
         return [
+          FormType.View({
+            View: Okta.View.extend({
+              template: '\
+              <span class="overflow-hidden" style="height:0; display:block" role="alert">{{alert}}</span>\
```

Comment:
Moved the styles to a class, and changed the security image to use that class as well.

The original request from the customer (https://oktainc.atlassian.net/browse/OKTA-114232) doesn't specify a type. I'm ok using "status" with the caveat that we might need to change it if the customer is not happy.